README.md

```diff
@@ -1,2 +1,132 @@
-# ruby-morse-decoder
-This project involves creating a Ruby program to decode Morse code messages, enabling the automated translation of Morse code characters and words into readable text. It promotes code reusability and simplifies the process of deciphering Morse code messages.
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
```

Comment:
- [ ] Great work 👏Kindly ensure to fix the broken logo link in the README for a professional documentation. Also, please ensure that you are not using the **template logo**. Optionally, you can use your own logo if you have one or you can simply remove it 🤓⛏

![image](https://github.com/Eccb7/ruby-morse-decoder/assets/106119105/2ff399c9-76cc-4cab-87b5-66378671c2bf)

Reply:
@KanzaTahreem  Done making changes